docs/docsource/operation/XDM_GUI/location_management/add_a_transient_source_storage_location.rst

```diff
@@ -0,0 +1,59 @@
+.. _Add a Transient Source Storage Location:
+
+Add a Transient Source Storage Location
+=======================================
+
+Adding a transient source storage location is quite similar to adding any other
+storage location, but for a few particulars.
+
+A transient source location is a temporary buffer to which data is stored and
+from which data is replicated. Scality RING with sproxyd is the only
+production-ready environment that supports the transient source replication
+feature (the |product| Local environment also supports this feature, but is suitable
+for testing purposes only). Data written to the transient source location can be
+replicated to any cloud service |product| supports.
+
+To deploy a transient source storage location:
+
+#. Click the **Storage Locations** button in the sidebar.
+
+#   .. image:: ../../Graphics/sidebar_storage_locations_button.png
```

Comment:
Image are not showing up in the github preview, not sure if intentional or if it works in sphinx though.

Reply:
If it builds, then the image links are sound (determining relevance is up to Team Human, however...)

Reply:
I think the `#` at the start of the line messes with the restructured `.. image` tag maybe.

Reply:
This file is not resolved yet, so don't sweat this detail.